http-clients/apache5-client/src/main/java/software/amazon/awssdk/http/apache5/Apache5HttpClient.java

```diff
@@ -452,12 +454,27 @@ public interface Builder extends SdkHttpClient.Builder<Apache5HttpClient.Builder
         Builder dnsResolver(DnsResolver dnsResolver);
 
         /**
+         * @deprecated this has been replaced with {{@link #tlsSocketStrategy(TlsSocketStrategy)}}
```

Comment:
Can we explain that this is here to ease migration from 4.5.x?

Reply:
done